### DISCUSSION TITLE
v7.0.0

### DISCUSSION THREAD

**Discussion by kayhantolga** [betalgo/openai#276] [Announcements]
### 7.0.0
- The code now supports .NET 7.0. Big cheers to @BroMarduk for making this happen.
- The library now automatically disposes of the Httpclient when it's created by the constructor. This feature is thanks to @BroMarduk.
- New support has been added for using more than one instance at the same time. Check out this [link](https://github.com/betalgo/openai/wiki/Working-with-Multiple-Instances) for more details. Thanks to @remixtedi for bringing this to my attention.
- A lot of small improvements have been done by @BroMarduk.
- **Breaking Changes** 😢
  - I've removed 'GPT3' from the namespace, so you might need to modify some aspects of your project. But don't worry, it's pretty simple! For instance, instead of writing `using OpenAI.GPT3.Interfaces`, you'll now write `using OpenAI.Interfaces`.
  - The order of the OpenAI constructor parameters has changed. It now takes 'options' first, then 'httpclient'.
    ```csharp
	//Before
	var openAiService = new OpenAIService(httpClient, options);
	//Now
	var openAiService = new OpenAIService(options, httpClient);
	```

## What's Changed
* Dispose HttpClient when created internally. by @BroMarduk in https://github.com/betalgo/openai/pull/243
* Fixes for Exception Warnings by @BroMarduk in https://github.com/betalgo/openai/pull/262
* Fixes for Warnings where CREFS are used instead of HREFS by @BroMarduk in https://github.com/betalgo/openai/pull/264
* DotNET 7 Project Support by @BroMarduk in https://github.com/betalgo/openai/pull/261
* Added support for multiple instance by @kayhantolga in https://github.com/betalgo/openai/pull/258
* removed GPT3 from namespace by @kayhantolga in https://github.com/betalgo/openai/pull/269
* nuget symbols by @kayhantolga in https://github.com/betalgo/openai/pull/272
* Version 7.0.0 by @kayhantolga in https://github.com/betalgo/openai/pull/271
* version bump for v7.0.0 by @kayhantolga in https://github.com/betalgo/openai/pull/275

## New Contributors
* @BroMarduk made their first contribution in https://github.com/betalgo/openai/pull/243

**Full Changelog**: https://github.com/betalgo/openai/compare/v6.8.6...v7.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/betalgo/openai/releases/tag/v7.0.0'>v7.0.0</a>.</em>

**Comment by Frogley** [comment]
Bravo!
